test/Specs/TextManipulationSpec.hs

```diff
@@ -36,3 +37,11 @@ spec = describe "Text manipulation" do
     it "chunksOfNonEmptyText strips chunks" $ do
       chunksOfNonEmptyText [compileNonEmptyTextKnownLength|a   b|]
         `shouldBe` ([widen [compileNonEmptyTextKnownLength|a|], widen [compileNonEmptyTextKnownLength|b|]] :: [NonEmptyText 2])
+
+  describe "nonEmptyChunksOfNonEmptyText" $ do
```

Comment:
just copycat of the above test cases. trying to think of anything that's edge-casier for this specific variation but still compiles.